### PR TITLE
Feature/id 29/event bus

### PR DESCRIPTION
**Pull request by Elvis-Castro-H**
## Pull Request
**Clickup:** https://app.clickup.com/t/868ac54av

## Reviewers
@victor-fundacion 
@JP-Arequipa

### CODE REVIEW

**Review by victor-fundacion — APPROVED** [review]
Approve.

File: apps/EventBus/EventBus.http [hunk 1]
```diff
@@ -0,0 +1,6 @@
+@EventBus_HostAddress = http://localhost:5233
+
+GET {{EventBus_HostAddress}}/weatherforecast/
```
Comment (JP-Arequipa) [inline apps/EventBus/EventBus.http]: /weatherforecast/ is the default example of swagger

File: pocs/SubscriberTest/Program.cs [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+// builder.Services.AddCors(options =>
+// {
+//     options.AddPolicy("AllowSpecificOrigin", builder =>
+//     {
+//         builder.WithOrigins("http://localhost:5233")
+//             .AllowAnyMethod()
+//             .AllowAnyHeader()
+//             .AllowCredentials();
+//     });
+// });
```
Comment (JP-Arequipa) [inline pocs/SubscriberTest/Program.cs]: commented code

**Comment by Elvis-Castro-H** [comment]
@JP-Arequipa all commentts were resolved

**Review by JP-Arequipa — APPROVED** [review]

**Pull request merged**